recipes/logr/0.1.0/conanfile.py

```diff
@@ -0,0 +1,92 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+class LogrConan(ConanFile):
+    name = "logr"
+    version = "0.1.0"
+    license = "BSD 3-Clause License"
+    homepage = "https://github.com/ngrodzitski/logr"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "Logger frontend substitution for spdlog, glog, etc for server/desktop applications"
+    topics = ("logger", "development", "util", "utils")
+    generators = "cmake"
+    settings = "os", "compiler", "build_type", "arch"
+    build_policy = "missing"
+    exports_sources = ["CMakeLists.txt"]
+
+    options = { 'spdlog_backend' : [True, False],
+                'glog_backend' : [True, False],
+                'log4cplus_backend' : [True, False] }
+
+    default_options = { 'spdlog_backend': True,
+                        'glog_backend': True,
+                        'log4cplus_backend': True }
+
```

Comment:
Can you please rename these options to: `spdlog`, `glog` and `log4cplus`.
Also can all 3 backends be used together, or are they to be used exclusive?

Reply:
The meaning of this options is whether the corresponding header file would be installed or not. By default all of them are present.